test/Succeed/EtaSingletonField.agda

```diff
@@ -0,0 +1,28 @@
+{-# OPTIONS --allow-unsolved-metas #-}
+module EtaSingletonField where
```

Comment:
Thanks for conjuring up this test case!

Concerning documentation: I would like to read in a testcase a brief documentation what the problem was, what the symptom, and what the expected new result should be.
I tried to fill this in here:
- #7184